course_catalog/management/commands/split_ocw_courses.py

```diff
@@ -0,0 +1,34 @@
+"""Management command for splitting OCW courses by run"""
+
+from django.core.management import BaseCommand
+from djoser.conf import settings
```

Comment:
Should this be `from django.conf import settings`?

Reply:
Yes, and I remember changing it yesterday afternoon, must have reverted again somehow, oops.  I'm going to blame PyCharm 😉